.chloggen/configopaque_stringer.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Given the impact you discovered in contrib, I would mark this as a `breaking` just to be safe and make sure more eyes are on this
```suggestion
change_type: breaking
```

---

For my to learn: Confused based on what golang rules this is breaking.

---

I guess Pablo is trying to make sure this is advertised as a change that can break some uses of the configopaque.String as part of its use with `fmt.Stringer`. This is not really a change impacting the go API.

---

> For my to learn: Confused based on what golang rules this is breaking.

Yes, arguably it's a breakage in behavior, but it's not impacting the API